### ISSUE TITLE
Onboarding: Next button disappears when the interface is too small. 

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#3656]
### Is your enhancement related to a problem? Please describe

When the interface / PD is too small, the next buttons disappear and I'm unable to find them unless I increase the size of Podman Desktop. 

https://github.com/containers/podman-desktop/assets/6422176/50d8e867-e05e-4298-bd4f-6592e54e6bbc

### Describe the solution you'd like

Being able to see the buttons regardless how small PD is.

**Comment by benoitf** [comment]
so should we have the next button floating or being on the upper side of the window ?

cc @mairin

**Comment by deboer-tim** [comment]
FWIW in the similar Welcome screen I'm pretty sure we went with fixed top and bottom controls and a scroll in the middle section (only when necessary).

**Comment by mairin** [comment]
👋🏻 We talked about this on the UX call this morning. We pretty much agreed the best solution would be to have the button bar at the bottom have a fixed positioning, so it moves with the window as it is resized. The onboarding content area above the button bar should be scrollable.

I also suggested having the button bar's background be partially transparent, to reinforce visually the notion that the content area is scrollable (eg in case they only see the scrollbar track when the content area is focused or they don't notice the scrollbar track.)

I could look something like this (i don't know that I'm 100% happy with the 75% opacity and the exact color value here so I will play with that and follow up):
![Screenshot from 2023-08-24 10-20-30](https://github.com/containers/podman-desktop/assets/799683/3e8d60f2-c6c9-472a-b555-ebed9bb73b01)
